### ISSUE TITLE
[Accessibility] Screen Reader announces editor multiple times

### ISSUE THREAD

**Issue by BennyHuang** [ckeditor/ckeditor4#5172]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open Demo page https://ckeditor.com/ckeditor-4/demo/ with chrome
2. Open screen reader NVDA or JAWS
3. Focus on editor

### Expected result

Screen Reader should announce one times

### Actual result

Screen Reader should announce multiple times
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/1995447/165594787-6fe01607-fb56-4b0a-8c4e-7502ad2748d2.png">

## Other details

* Browser: Chrome 100
* OS: Windows 10
* CKEditor version: 4.17.1

**Comment by Comandeer** [comment]
Duplicate of #2445.

**Comment by jacekbogdanski** [comment]
@BennyHuang it seems like this issue is actually not a duplicate of #2445 as a fix for it has been targeting a very similar problem. However, we will be tracking that bug in https://github.com/ckeditor/ckeditor4/issues/5185, so please subscribe to that ticket if you would like to get updates on our progress on it.
